docs/content/rn-intro-con.adoc

```diff
@@ -1,6 +1,11 @@
+[id='concept-explanation-{context}']
+= Overview of {PRODUCT}
 
-[id='rn-intro-con']
+{PRODUCT} enables your team to develop, build and deploy enterprise integration solutions on a private Kubernetes cluster managed by Red Hat. 
+{PRODUCT} enables developers adopt event-based computing and API-based development with lightweight, loosely coupled, self-service and flexible components. 
+{PRODUCT} enables operations focus on high value objectives because Red Hat manage the infrastructure and everything is protected by RHSSO.
```

Comment:
I would recommend removing this last sentence, I don't think it adds anything but more confusing detail on the "managed by Red Hat" for initial user understanding of the service.  I would expect the layer of detail of what it saves Operations to be captured in a more detailed area of the documentation.